syntax = "proto3";

option java_multiple_files = true;
option java_package = "lis.gcv.chromosomesearch";
option java_outer_classname = "ChromosomeSearchProto";
option objc_class_prefix = "GS";

package chromosomesearch;


service ChromosomeSearch {
  rpc Search (SearchRequest) returns (SearchReply) {}
}


message SearchRequest {
  string query = 1;
}


message SearchReply {
  repeated string chromosomes = 1;
}
